normalizer/tests/test_jageocoder.py

```
@@ -0,0 +1,220 @@
+    if coords:
+        print("\n逆ジオコーディング結果:")
+        for api_name, (lat, lon) in coords.items():
+            print(f"\n{api_name}の座標({lat:.6f}, {lon:.6f})から:")
+            
+            # Google逆引き
+            if api_key:
+                reverse_addr = reverse_geocode_google(lat, lon, api_key)
+                if reverse_addr:
+                    print(f"  Google逆引き: {reverse_addr}")
+                else:
+                    print(f"  Google逆引き: [NG] 取得失敗")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)